### PR TITLE
Fix basic vision example

### PR DESCRIPTION
**Pull request by williamhaley**
Related to https://github.com/wpilibsuite/WPILibPi/issues/171 for https://docs.wpilib.org/en/latest/docs/software/vision-processing/frcvision/basic-vision-example.html

Fixing some basic import/naming errors and swiping some sample code from https://robotpy.readthedocs.io/en/stable/vision/code.html to get things running.

But also, I'm just starting off with Vision so I can't properly validate that these changes are ideal or fully working. I can keep at it to try and see some meaningful results, but figured someone more knowledgeable could take a look at this in case it helps as a jumping off point.

I have to admit, I'm not 100% clear on whether or not the intention of this example code is that it can be readily upload-able to the FRCVision-rPi web Application UI or if the intention is more of a jumping-off point for teams. That `main()` may not be appropriate if other libraries are meant to invoke this.

### CODE REVIEW

**Comment by Daltz333** [comment]
There are broken things in the wpilibpi image for the python program right now (pycore is not compiled for rasberrypi). Marked as waiting on wpilib.

File: source/docs/software/vision-processing/frcvision/basic-vision-example.rst [hunk 1]
```diff
@@ -13,9 +13,11 @@ This is an example of a basic vision setup that posts the target's location in t
       import cv2
       import json
       import numpy as np
+      import time
 
       def main():
-         config = json.load('/boot/frc.json')
+         f = open('/boot/frc.json')
+         config = json.load(f)
```
Comment (auscompgeek) [inline source/docs/software/vision-processing/frcvision/basic-vision-example.rst:20]: As is, this file is never closed.
```suggestion
         with open('/boot/frc.json') as f:
             config = json.load(f)
```

**Comment by jasondaming** [comment]
@williamhaley it looks like the mentioned issue has been closed so I think we are clear to make more progress on this

**Comment by williamhaley** [comment]
@jasondaming Let me know how best to proceed on this

**Comment by Daltz333** [comment]
1. A rebase off of master
2. Address @auscompgeek comment

File: source/docs/software/vision-processing/frcvision/basic-vision-example.rst (hunk 1 above)
Comment (williamhaley) [reply source/docs/software/vision-processing/frcvision/basic-vision-example.rst]: 👍

**Comment by williamhaley** [comment]
✔️ Updates pushed

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**